.github/workflows/release.yml

```diff
@@ -139,10 +139,37 @@ jobs:
         run: az acr login --name ${{ env.ACR_REPO }}
 
       - name: Push Docker Image
-        run: docker push ${{ env.ACR_REPO }}/${{ env.PREFIX }}:latest
+        run: docker push ${{ env.ACR_REPO }}/${{ env.PREFIX }} --all-tags
+
+      - name: Fetch GitHub Action Public IP
+        id: ip
+        uses: haythem/public-ip@v1.2
+
+      - name: Whitelist GitHub Action Running in Function App Slot
```

Comment:
A whitelist is needed because we can't setup the function app with a private endpoint (that supports the VPN) or the Front Door will not be able to connect.